src/app/app-routing.module.ts

```diff
@@ -78,6 +79,11 @@ export const ROUTES: DrawerItem[] = [
                 path: 'sectioned-form',
                 component: SectionedFormComponent,
             },
+            {
+                title: 'Phone Number Format',
+                path: 'phone-number-format',
+                component: PhoneNumberFormatComponent,
+            },
```

Comment:
Can we move this up so that the DrawerNavItems are in the same order as the designs?